"Hello Microverse" Project

DebyGrey opened the pull request:
## Hello Microverse: first project

In this project, I:

- Added an index.html file that includes the header, "Hello Microverse!"
- Added a style.css file that includes a color style for the header
- Added a test.md file that should be ignored by git
- Set up the linters for HTML&CSS

franclobo reviewed (commented):
Hi @DebyGrey, 

![terminator](https://user-images.githubusercontent.com/58642949/226769623-794e12c0-bce7-4cec-9f73-7de3f8d0f305.gif)

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have a descriptive PR with a meaningful title and a summary of the actions.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

DebyGrey commented:
Hello @franclobo,

Thank you for taking out time to review my project, I apologize for the inconvenience, and I have added a descriptive PR, meaningful title, and summary of actions as requested.

However, I have some concerns about my PR. I just noticed the linter checker is not activated on my end and that leaves me confused about whether to make another pull request because the linter checker activates once a pull request is made.

Please, I need your advice on what to do.

Thanks

franclobo commented:
> Hello @franclobo,
> 
> Thank you for taking out time to review my project, I apologize for the inconvenience, and I have added a descriptive PR, meaningful title, and summary of actions as requested.
> 
> However, I have some concerns about my PR. I just noticed the linter checker is not activated on my end and that leaves me confused about whether to make another pull request because the linter checker activates once a pull request is made.
> 
> Please, I need your advice on what to do.
> 
> Thanks

Hi @DebyGrey please 🙏🏽 review the name of your folder the correct name is `.github/workflows` instead of `.github/workflow`

DebyGrey commented:
> > Hello @franclobo,
> > Thank you for taking out time to review my project, I apologize for the inconvenience, and I have added a descriptive PR, meaningful title, and summary of actions as requested.
> > However, I have some concerns about my PR. I just noticed the linter checker is not activated on my end and that leaves me confused about whether to make another pull request because the linter checker activates once a pull request is made.
> > Please, I need your advice on what to do.
> > Thanks
> 
> Hi @DebyGrey please 🙏🏽 review the name of your folder the correct name is `.github/workflows` instead of `.github/workflow`

Hi @franclobo I have corrected it and pushed

DebyGrey commented:
> > > Hello @franclobo,
> > > Thank you for taking out time to review my project, I apologize for the inconvenience, and I have added a descriptive PR, meaningful title, and summary of actions as requested.
> > > However, I have some concerns about my PR. I just noticed the linter checker is not activated on my end and that leaves me confused about whether to make another pull request because the linter checker activates once a pull request is made.
> > > Please, I need your advice on what to do.
> > > Thanks
> > 
> > 
> > Hi @DebyGrey please 🙏🏽 review the name of your folder the correct name is `.github/workflows` instead of `.github/workflow`
> 
> Hi @franclobo I have corrected it and pushed

Also @franclobo, I just noticed my linter checker is back up but the lighthouse still didn't pass the check and I don't know why. Can you please recommend material for this issue, please keep in mind the operating system is mac os.

DebyGrey commented:
> > > > Hello @franclobo,
> > > > Thank you for taking out time to review my project, I apologize for the inconvenience, and I have added a descriptive PR, meaningful title, and summary of actions as requested.
> > > > However, I have some concerns about my PR. I just noticed the linter checker is not activated on my end and that leaves me confused about whether to make another pull request because the linter checker activates once a pull request is made.
> > > > Please, I need your advice on what to do.
> > > > Thanks
> > > 
> > > 
> > > Hi @DebyGrey please 🙏🏽 review the name of your folder the correct name is `.github/workflows` instead of `.github/workflow`
> > 
> > 
> > Hi @franclobo I have corrected it and pushed
> 
> Also @franclobo, I just noticed my linter checker is back up but the lighthouse still didn't pass the check and I don't know why. Can you please recommend material for this issue, please keep in mind the operating system is mac os.

Hi @franclobo, I have successfully fixed the lighthouse issue. You may go ahead with the second review. Thank you

AbelG101 commented on README.md:
- [x] Please make sure to remove any section descriptions from the readme as they are only there to guide you (the project owner) on what to place on this section and should be removed in the final version 👍

   Kindly remove this and place a description of the project:
   - > Describe your project in 1 or 2 sentences.
- [x] Kindly as I mentioned previously all section descriptions should be removed from your readme file as they are only there to guide you (the project owner) on what to place on this section and should be removed in the final version 👍.
Kindly remove this:
  - > Describe the tech stack and include only the relevant sections that apply to your project
- [x] Please make sure to only add the tech stack that you've used for the project. So please replace the default tech stacks listed in the tech stack section with what you've used like HTML5 & CSS3
- [x] Please make sure to add key features to your project, it could be as simple as "displays Hello Microverse" 📌 as it gives a good overview to the reader of what to look at. I noticed you removed this section but please know that you can only remove the "FAQ" section and "Live demo" link section (and this is only if the project doesn't require a live demo link)
- [x] Please also make sure to remove any link that doesn't work from the table of contents, I noticed the Live demo link doesn't work as you don't have that section that's okay but also make sure to remove the link from the list of contents
- [x] Kindly make sure to add clear and detailed steps on how to get started with your project such as the prerequisites, setting up, etc... the default text should be removed 🎯
- [x]  Kindly as I mentioned previously all section descriptions should be removed from your readme file as they are only there to guide you (the project owner) on what to place on this section and should be removed in the final version 👍.
Kindly remove this:
   - > Mention all of the collaborators of this project.
- [x] Kindly as I mentioned previously all the main headlines should exist in your readme as I've noticed you've also removed the "Future Features section" make sure to add it with at least 1 feature you'll be adding this shows future clients you've got a growth mindset by continuously adding features to your project👍.
- [x] Kindly as I mentioned previously all section descriptions should be removed from your readme file as they are only there to guide you (the project owner) on what to place on this section and should be removed in the final version 👍.
Kindly remove this:
    - > Write a message to encourage readers to support your project
- [x]  Kindly as I mentioned previously all section descriptions should be removed from your readme file as they are only there to guide you (the project owner) on what to place on this section and should be removed in the final version 👍.
Kindly remove this:
    - > Give credit to everyone who inspired your codebase.
- [x] This [MIT](https://github.com/DebyGrey/Hello-microverse/blob/first-project/LICENSE) link is broken so please make sure to fix it as it shows this project has an appropriate license. The reason it doesn't work is that you don't have an MIT.md file on your repo so please add that file (you can find it [here](https://github.com/microverseinc/readme-template/blob/master/MIT.md)) and modify the content inside that file with your personal info, and finally add that file as a link like this: `./MIT.md` to your MIT link.
- [x] Kindly as I mentioned previously all section descriptions should be removed from your readme file as they are only there to guide you (the project owner) on what to place on this section and should be removed in the final version 👍.
Kindly remove this:
    -  _NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._

AbelG101 reviewed (changes requested):
![](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExNzFmY2U0ZTkwMmM3ZTM5MDBkNjhhZjk0MjI0OTU3MGNjYjMyMmRhOCZjdD1n/M1fddM48g9084bJMEX/giphy.gif)


## Hi @DebyGrey 👋, 

You're doing a good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Correct use of GitHub Flow ✔
-  Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

## Required Changes ♻️
- _Check the comments under the review._
- You're doing a really great job 🥇making the changes requested previously but it's the mark of a great developer to provide a good description of their project, you seem to be missing a few things in your readme but you're almost done. So stay strong and make the changes I've requested 💪. Good luck!

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

DebyGrey commented:
> ![](https://camo.githubusercontent.com/533b1c3f9cb7f5df7437c8cd2337c23b3834a305fefe709ce76f950be34133d1/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f76312e59326c6b505463354d4749334e6a45784e7a466d593255305a546b774d6d4d335a544d354d44426b4e6a68685a6a6b304d6a49304f5455334d474e6a596a4d794d6d52684f435a6a6444316e2f4d316664644d34386739303834624a4d45582f67697068792e676966) [ ![68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f76312e59326c6b505463354d4749334e6a45784e7a466d593255305a546b774d6d4d335a544d354d44426b4e6a68685a6a6b304d6a49304f5455334d474e6a596a4d794d6d52684f435a6a6444316e2f4d316664644d34386739303834624a4d45582f67697068792e676966](https://camo.githubusercontent.com/533b1c3f9cb7f5df7437c8cd2337c23b3834a305fefe709ce76f950be34133d1/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f76312e59326c6b505463354d4749334e6a45784e7a466d593255305a546b774d6d4d335a544d354d44426b4e6a68685a6a6b304d6a49304f5455334d474e6a596a4d794d6d52684f435a6a6444316e2f4d316664644d34386739303834624a4d45582f67697068792e676966) ](https://camo.githubusercontent.com/533b1c3f9cb7f5df7437c8cd2337c23b3834a305fefe709ce76f950be34133d1/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f76312e59326c6b505463354d4749334e6a45784e7a466d593255305a546b774d6d4d335a544d354d44426b4e6a68685a6a6b304d6a49304f5455334d474e6a596a4d794d6d52684f435a6a6444316e2f4d316664644d34386739303834624a4d45582f67697068792e676966) [ ](https://camo.githubusercontent.com/533b1c3f9cb7f5df7437c8cd2337c23b3834a305fefe709ce76f950be34133d1/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f76312e59326c6b505463354d4749334e6a45784e7a466d593255305a546b774d6d4d335a544d354d44426b4e6a68685a6a6b304d6a49304f5455334d474e6a596a4d794d6d52684f435a6a6444316e2f4d316664644d34386739303834624a4d45582f67697068792e676966)
> 
> ## Hi @DebyGrey 👋,
> You're doing a good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
> 
> ## Highlights
> * Correct use of GitHub Flow ✔
> * Well-structured files ✔
> * All linter checks are passing ✔
> * Descriptive PR title with a great summary ✔
> 
> ## Required Changes ♻️
> * _Check the comments under the review._
> * You're doing a really great job 🥇making the changes requested previously but it's the mark of a great developer to provide a good description of their project, you seem to be missing a few things in your readme but you're almost done. So stay strong and make the changes I've requested 💪. Good luck!
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
> 
> You can also consider:
> 
> * N/A
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Hi, @AbelG101 @franclobo I have made the changes where necessary. Kindly review my project.

Thank you

DebyGrey commented:
> > ![](https://camo.githubusercontent.com/533b1c3f9cb7f5df7437c8cd2337c23b3834a305fefe709ce76f950be34133d1/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f76312e59326c6b505463354d4749334e6a45784e7a466d593255305a546b774d6d4d335a544d354d44426b4e6a68685a6a6b304d6a49304f5455334d474e6a596a4d794d6d52684f435a6a6444316e2f4d316664644d34386739303834624a4d45582f67697068792e676966) [ ![68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f76312e59326c6b505463354d4749334e6a45784e7a466d593255305a546b774d6d4d335a544d354d44426b4e6a68685a6a6b304d6a49304f5455334d474e6a596a4d794d6d52684f435a6a6444316e2f4d316664644d34386739303834624a4d45582f67697068792e676966](https://camo.githubusercontent.com/533b1c3f9cb7f5df7437c8cd2337c23b3834a305fefe709ce76f950be34133d1/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f76312e59326c6b505463354d4749334e6a45784e7a466d593255305a546b774d6d4d335a544d354d44426b4e6a68685a6a6b304d6a49304f5455334d474e6a596a4d794d6d52684f435a6a6444316e2f4d316664644d34386739303834624a4d45582f67697068792e676966) ](https://camo.githubusercontent.com/533b1c3f9cb7f5df7437c8cd2337c23b3834a305fefe709ce76f950be34133d1/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f76312e59326c6b505463354d4749334e6a45784e7a466d593255305a546b774d6d4d335a544d354d44426b4e6a68685a6a6b304d6a49304f5455334d474e6a596a4d794d6d52684f435a6a6444316e2f4d316664644d34386739303834624a4d45582f67697068792e676966) [ ](https://camo.githubusercontent.com/533b1c3f9cb7f5df7437c8cd2337c23b3834a305fefe709ce76f950be34133d1/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f76312e59326c6b505463354d4749334e6a45784e7a466d593255305a546b774d6d4d335a544d354d44426b4e6a68685a6a6b304d6a49304f5455334d474e6a596a4d794d6d52684f435a6a6444316e2f4d316664644d34386739303834624a4d45582f67697068792e676966)
> > ## Hi @DebyGrey 👋,
> > You're doing a good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
> > ## Highlights
> > 
> > * Correct use of GitHub Flow ✔
> > * Well-structured files ✔
> > * All linter checks are passing ✔
> > * Descriptive PR title with a great summary ✔
> > 
> > ## Required Changes ♻️
> > 
> > * _Check the comments under the review._
> > * You're doing a really great job 🥇making the changes requested previously but it's the mark of a great developer to provide a good description of their project, you seem to be missing a few things in your readme but you're almost done. So stay strong and make the changes I've requested 💪. Good luck!
> > 
> > ## Optional suggestions
> > _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
> > You can also consider:
> > 
> > * N/A
> > 
> > Cheers and Happy coding!👏👏👏
> > Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> > **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> > _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
> 
> Hi, @AbelG101 @franclobo I have made the changes where necessary. Kindly review my project.
> 
> Thank you

@franclobo @AbelG101 Please hold with the review, I need help. My MIT license link is still broken. How can I fix that?

DebyGrey commented:
Hello @AbelG101 @franclobo, I have figured out how to add the MIT.md link and have made changes to my README.md file as requested. Please you can go ahead and review my project.

Thank you

TuyishimireEric commented on README.md:
```diff
@@ -27,44 +26,48 @@
 <!-- GETTING STARTED -->
 
 ## 💻 Getting Started <a name="getting-started"></a>
 
-> Describe how a new developer could make use of your project.
 
 To get a local copy up and running, follow these steps.
 
 ### Prerequisites
 
 In order to run this project you need:
+ A web browser, google chrome preferably.
+ A code editor (e.g vscode) 
 
 
 ### Setup
```
- [x] **Good job so far 👏 👏**, Kindly as suggested by the previous reviewer, I think it would be better to remove these default steps on how to get started, This one of the most important part in a README as it helps the user to use your project, 🙏 

 ##### For Example:

### Prerequisites

In order to run this project you need:
To clone or fork and run it in a browser

<!--
Example command:

```sh
 gem install rails
```
 -->

### Setup

Clone this repository to your desired folder:




```sh
  cd my-folder
  git clone https://github.com/Paulinelali/hello-microverse-project
```
---
You may try to give the guides to the user to get started 🙏

TuyishimireEric reviewed (changes requested):
# Status: REQUIRED CHANGES :recycle: 
### Hi  @DebyGrey    👋 👋 
#### I'm @TuyishimireEric, Your code reviewer,
Good job so far! 🥇 
There are some issues that you still need to work on to go to the next project but you are almost there!
![](https://camo.githubusercontent.com/6ac905aa3a075ef45de55758c20c6e6efb1dfdbbb0d9fd03d4718d26c5c96cde/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966)
## Highlight 
✔ Used correct GitFlow👍 
✔ No Linter Errors 👍 

## Required Changes ♻️

- [ ]  Check the comments below 

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

DebyGrey commented:
> # Status: REQUIRED CHANGES ♻️
> ### Hi @DebyGrey 👋 👋
> #### I'm @TuyishimireEric, Your code reviewer,
> Good job so far! 🥇 There are some issues that you still need to work on to go to the next project but you are almost there! ![](https://camo.githubusercontent.com/6ac905aa3a075ef45de55758c20c6e6efb1dfdbbb0d9fd03d4718d26c5c96cde/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966)
> 
> ## Highlight
> ✔ Used correct GitFlow👍 ✔ No Linter Errors 👍
> 
> ## Required Changes ♻️
> * [x]   Check the comments below
> 
> **Cheers and Happy coding!👏👏👏**
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

hi, @TuyishimireEric I have made the changes as requested. Please review my project

thanks

TuyishimireEric commented:
Kindly, Submit your project again

DebyGrey commented:
> Kindly, Submit your project again

@TuyishimireEric Just did. Good morning

AbelG101 reviewed (approved):
![](https://media.giphy.com/media/YPI7uEvEZr7siYRlZD/giphy.gif)

## Hi @DebyGrey  👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it. 
Congratulations! 🎉 

## Highlights
- Correct use of GitHub Flow ✔
-  Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

DebyGrey commented:
Yes! Thank you! 😁😁😁😁😁😁

On Thu, Mar 23, 2023 at 8:46 AM Abel Gebeyehu ***@***.***>
wrote:

> ***@***.**** approved this pull request.
>
>
> <https://camo.githubusercontent.com/13011b27700826e41ed2d653d44c281f02454f2c069ad79b3dbc4a84ad7bf22b/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f59504937754576455a7237736959526c5a442f67697068792e676966>
> Hi @DebyGrey <https://github.com/DebyGrey> 👋,
>
> Your project is complete! There is nothing else to say other than... it's
> time to merge it.
> Congratulations! 🎉
> Highlights
>
>    - Correct use of GitHub Flow ✔
>    - Well-structured files ✔
>    - All linter checks are passing ✔
>    - Descriptive PR title with a great summary ✔
>
> Optional suggestions
>
>    - N/A
>
> *Every comment with the [OPTIONAL] prefix won't stop the approval of this
> PR. However, I strongly recommend you take them into account as they can
> make your code better. Some of them were simply missed by the previous
> reviewer and addressing them will really improve your application.*
>
> Cheers and Happy coding!👏👏👏
>
> Feel free to leave any questions or comments in the PR thread if something
> is not 100% clear.
> Please, remember to tag me in your question so I can receive the
> notification.
> ------------------------------
>
> *As described in the Code reviews limits policy
> <https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have a
> limited number of reviews per project (check the exact number in your
> Dashboard). If you think that the code review was not fair, you can request
> a second opinion using this form <https://airtable.com/shrQAqnBwek5a0O0s>.*
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/DebyGrey/Hello-microverse/pull/1#pullrequestreview-1354011523>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALYG7ZC7KVSCTTEWTP4MV7LW5P5WZANCNFSM6AAAAAAWDD66DY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
